Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -558,6 +558,23 @@
       components:
       - SurplusBundle
 
+- type: listing
+  id: uplinkHotPotato
+  name: uplink-hot-potato-name
+  description: uplink-hot-potato-desc
+  productEntity: HotPotato
+  cost:
+    Telecrystal: 4
+  categories:
+  - UplinkJob
+  conditions:
+  - !type:BuyerJobCondition
+    whitelist:
+    - Chef
+    - Botanist
+    - Clown
+    - Mime
```

Comment:
?

Reply:
I took the idea from tgstation so it should make sense
![image](https://user-images.githubusercontent.com/40753025/220892990-855c713a-a465-4842-835a-486d98e92b82.png)